toolchain/sem_ir/name_scope.h

```diff
@@ -18,13 +18,101 @@ enum class AccessKind : int8_t {
   Private,
 };
 
+// Represents the result of a name lookup.
+// A lookup results are constructor through the `Make()` factory functions. The
```

Comment:
I think here you mean one of:

```suggestion
// Lookup results are constructed through the `Make()` factory functions. The
```

```suggestion
// A lookup results is constructed through the `Make()` factory functions. The
```

Reply:
Done.